index.js

```diff
@@ -20,8 +20,10 @@ module.exports = {
     'plugin:jest/recommended',
     'plugin:jest-formatting/strict',
     'plugin:react-hooks/recommended',
+    'plugin:import/typescript',
+    'plugin:typescript-enum/recommended',
   ],
-  plugins: ['backpack', 'prettier', 'jest-formatting'],
+  plugins: ['backpack', 'prettier', 'jest-formatting', 'typescript-enum'],
```

Comment:
The README of https://github.com/shian15810/eslint-plugin-typescript-enum matches our reasoning here. 

TL;DR we only wish to turn on TypeScript type annotations, but restrict its language expression features. This plugin allows us to disallow the use of `enums`.

https://www.typescriptlang.org/docs/handbook/enums.html